### PR TITLE
sorting the left column cards by card type. also fixing the battlefields

### PR DESCRIPTION
**Pull request by SirZach**

### CODE REVIEW

File: tests/unit/components/game/battle-field-test.js [hunk 1]
```diff
@@ -12,7 +12,9 @@ test('it renders', function(assert) {
   assert.expect(2);
 
   // creates the component instance
-  var component = this.subject();
+  var component = this.subject({
+    nonLandCards: []//this computed property breaks during CL tests but not browser tests :(
```
Comment (ahaurw01) [inline tests/unit/components/game/battle-field-test.js]: I think this is because phantom doesn't implement `Function.prototype.bind`.

Comment (SirZach) [inline tests/unit/components/game/battle-field-test.js]: benoit balls.

Comment (ahaurw01) [inline tests/unit/components/game/battle-field-test.js]: yeah

**Comment by ahaurw01** [comment]
Looks great! It seems like the tapping motion is no longer fluid. I think the dom elements are getting ripped out and replaced for some reason. If the reason jumps out at you, awesome. Otherwise I say merge and we can address later. Not jumping out at me at the moment.

**Comment by SirZach** [comment]
so the non-land cards still animate just fine for me but the land cards are really jumpy. it's must have to do with the updates in this branch. i'm going to merge this branch in so we get the separation in but i'll explore more with it. i think if we get that observer in to set the cards instead of it being on computed properties a lot of these problems will go away

**Pull request merged**